File: Striver/A-Z/Step 1/switch_case.cpp
#include<iostream>
using namespace std;
#include<vector>
#include <math.h>

double areaSwitchCase(int ch, vector<double> a) {

	switch(ch){
		case 1:
			return M_PI * a[0] * a[0];
		case 2:
			return a[0] * a[1];
	}
    return -1;
}

int main(){
    vector<double> d;
    d.push_back(10);
    cout << areaSwitchCase(1, d) << endl;
}

File: Striver/A-Z/Step 1/Python/switch_case.py
import math
from typing import *

def areaSwitchCase(ch: int, a: List[float]):
    # Write your code here
    ans = -1
    match ch:
        case 1:
            return '{:.5f}'.format(math.pi * a[0] * a[0])
        case 2:
            return '{:.5f}'.format(a[0] * a[1])
        

## Formatting the Number was a new thing I came to know.